deploy/jbossas/modeshape-jbossas-subsystem/src/main/java/org/modeshape/jboss/subsystem/AddIndexProvider.java

```diff
@@ -40,7 +41,13 @@
 
     public static final AddIndexProvider INSTANCE = new AddIndexProvider();
 
-    private static final Logger LOG = Logger.getLogger(AddIndexProvider.class.getPackage().getName());
+    private static final Map<String, String> MODULE_NAMES_BY_PROVIDER_ALIAS = new HashMap<String, String>() {
+        {
+            put("org.modeshape.jcr.index.lucene.LuceneIndexProvider", "org.modeshape.index-provider.lucene");
+            put("org.modeshape.jcr.index.elasticsearch.EsIndexProvider", "org.modeshape.index-provider.elasticsearch");
+            put("org.modeshape.jcr.index.local.LocalIndexProvider", "org.modeshape");
```

Comment:
Wouldn't it be useful to _also_ add aliases such as `lucene`, `elasticsearch`, and `local`? Seems like those simple aliases would be easier for people to use correctly.